cmd/cert-checker/main.go

```diff
@@ -91,10 +91,9 @@ func newChecker(saDbMap certDB, clk clock.Clock, pa core.PolicyAuthority, period
 	return c
 }
 
-const (
-	getCertsCountQuery = "SELECT count(*) FROM certificates WHERE issued >= :issued AND expires >= :now"
-	getCertsQuery      = "SELECT * FROM certificates WHERE issued >= :issued AND expires >= :now AND serial > :lastSerial LIMIT :limit"
-)
+const getCertsCountQuery = "SELECT count(*) FROM certificates WHERE issued >= :issued AND expires >= :now"
+
+var getCertsQuery = fmt.Sprintf("SELECT %s FROM certificates WHERE issued >= :issued AND expires >= :now AND serial > :lastSerial LIMIT :limit", sa.CertificateFields)
```

Comment:
This query doesn't actually get passed through `safeSelectOne`. Probably we should move this query to be adjacent to where it gets executed, so it's obvious that the interpolation is safe.